docs/Formhandling.stories.mdx

```diff
@@ -0,0 +1,178 @@
+    setCountry(e.detail.selectedOption.innerText);
+  };
+  const handleBirthdayChange = (e) => {
+    setBirthday(e.detail.value);
+  };
+  const handlePaymentSelectionChange = (e) => {
+    const selected = [];
+    Object.entries(e.detail.items).forEach(([key, val]) => {
+      selected.push(val.text);
+    });
+    setPayment(selected);
+  };
+  const handleTermsChange = (e) => {
+    setTerms(e.target.checked);
+  };
+
+  return (
+    <ThemeProvider>
+      <Form onSubmit={handleSubmit}>
+        <FormGroup titleText="Create Account">
+          <FormItem label={<Label required>Email</Label>}>
+            <Input name="email" required type={InputType.Email} value={email} onInput={handleEmailInput} />
+          </FormItem>
+          <FormItem label={<Label required>Password</Label>}>
+            <Input name="password" required type={InputType.Password} value={password} onInput={handlePasswordInput} />
+          </FormItem>
+          <FormItem label="Country">
+            <Select name="country" onChange={handleCountryChange}>
+              <Option>Germany</Option>
+              <Option>France</Option>
+              <Option>Italy</Option>
+            </Select>
+          </FormItem>
+          <FormItem label="Date of Birth">
+            <DatePicker value={birthday} onChange={handleBirthdayChange}></DatePicker>
+          </FormItem>
+          <FormItem label="Payment methods">
+            <MultiComboBox onSelectionChange={handlePaymentSelectionChange}>
+              <MultiComboBoxItem text="Credit card" />
+              <MultiComboBoxItem text="PayPal" />
+              <MultiComboBoxItem text="Bank transfer" />
+            </MultiComboBox>
+          </FormItem>
+          <FormItem>
+            <CheckBox
+              required
+              name="terms"
+              text="I accept the terms of service"
+              checked={terms}
+              onChange={handleTermsChange}
+            />
+          </FormItem>
+          <FormItem>
+            <Button submits>Submit</Button>
+          </FormItem>
+        </FormGroup>
+      </Form>
+    </ThemeProvider>
+  );
+}
+```
+
+</details>
+
+Now the `RegisterForm` would compile to this:
+
+<RegisterForm />
+
+For the submits property of the `Button` to have effect, you must add the following import to your project: `import '@ui5/webcomponents/dist/features/InputElementsFormSupport.js';`
+
+Whenever a component has a custom event (e.g. onChange event from `DatePicker`), the corresponding value should be fetched via event.detail (not event.target). This will give you the correct updated value since the internal input has already been updated. At the time the event is fired, however, it is not yet updated and therefore event.target might also not be updated yet.
+
+By default, the browser will reload the page when a form submission event is triggered. Generally, you want to avoid this in React.js applications because it would cause you to lose the state.
```

Comment:
```suggestion
By default, the browser will reload the page when a form submission event is triggered. Generally, you want to avoid this in React applications because it would cause you to lose the state.
```